### PR TITLE
add eol notification for openshift pipeline plugin

### PR DESCRIPTION
**Pull request by gabemontero**
@openshift/sig-developer-experience @bmcelvee fyi / ptal

### CODE REVIEW

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -844,7 +844,10 @@ for more information.
 
 The {product-title} Pipeline Plug-in is a prior integration between Jenkins and
 {product-title} which provides less functionality than the {product-title}
-Client Plug-in. It remains available and supported.
+Client Plug-in. It remains available and supported to work with {product-title}
+versions up to v3.11.  For later verions of {product-title}, either use the
+`oc` binary directly from your Jenkins Pipelines, or use the 
+xref:#client-plug-in[{product-title} Client Plug-in].
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: I would change "it remains available and supported" to "It has been deprecated but continues to work with..."

**Comment by bparees** [comment]
heh, you beat me to it :)

File: using_images/other_images/jenkins.adoc (hunk 1 above)
Comment (gabemontero) [reply using_images/other_images/jenkins.adoc]: update pushed

**Comment by bparees** [comment]
lgtm.  We will still want a release note both in 3.11 (warning people it's going away soon) and 4.0 (making it clear it's not supported anymore)

**Comment by vikram-redhat** [comment]
@bparees - do we need to remove the docs for pipelines from 4.0? For example: https://docs.openshift.com/container-platform/3.10/dev_guide/dev_tutorials/openshift_pipeline.html

**Comment by bparees** [comment]
@vikram-redhat no, this is just about a single openshft integration plugin which was poorly named, the overall pipeline feature is unaffected.

**Pull request merged**